packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row_default.story.vue

```diff
@@ -159,29 +159,3 @@ export default {
   },
 };
 </script>
-
-<style lang="less" scoped>
-  .feed-item-row {
-    &__default-story {
-      p {
-        color: var(--dt-color-foreground-primary);
-        font-size: 15px;
-        font-style: normal;
-        font-weight: 400;
-        line-height: 24px;
-      }
-    }
-    &__thread{
-      padding-top: .3rem;
-      padding-bottom: .3rem;
-    }
-    &__reply{
-      color: var(--dt-color-foreground-tertiary);
-      font-size: 12px;
-      font-style: normal;
-      font-weight: 400;
-      line-height: 16px;
-      letter-spacing: -0.12px;
-    }
-  }
-</style>
```

Comment:
All this styling makes it very confusing to know what the component is doing and what the example is doing.. removed.